src/metabase/driver.clj

```diff
@@ -33,13 +34,28 @@
       (catch Throwable e
         (log/error e (trs "Failed to notify {0} Database {1} updated" driver id))))))
 
+(defn- short-timezone-name [timezone-id]
```

Comment:
Moving these functions & settings into `metabase.driver` and `metabase.driver.util` is also to resolve a (quite convoluted) circular dependency. But I think the new locations make sense.